CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -509,6 +509,50 @@ function PlayerHospital:makeRaiseRequest(amount, staff)
   self.world.ui.bottom_panel:queueMessage("strike", amount, staff)
 end
 
+--! Announce to the player that a staff member is leaving the hospital
+--!param staff (table) The staff member
+function PlayerHospital:announceStaffLeave(staff)
+  self.world.ui:playRandomAnnouncement(staff.leave_sounds, staff.leave_priority)
+end
+
+--! Advise the player that they attempted to put a staff member in a room they can't attend to
+--!param room (table)
+--!param staff_member (table)
+function PlayerHospital:adviseWrongStaffForRoom(room, staff_member)
```

Comment:
A handyman definitely can't fall into this function? Looks like it from room.lua:317 .

Reply:
The end of that if returns, so handymen aren't included in the check for all staff.